ExperimentConfigSidecar/Models/ConfigurationEvent.cs

```diff
@@ -0,0 +1,13 @@
+namespace ExperimentConfigSidecar.Models;
+
+/// <summary>
+/// Event to signal a configuration change.
```

Comment:
Maybe also indicate that this event originates from the experiment-config service

Reply:
changed with 13fb59a8872aa2042202e017d1fd30d1b29144be